files/base.rb

```diff
@@ -63,6 +63,24 @@ def team_data(lookup_key = nil)
     data
   end
 
+  def check_data(lookup_key = nil)
```

Comment:
Gotcha. Looking at the tests, it looks like the only reason we have that is to protect against the situation where we don't have a key (like a team with no hipchat default room, lack of a team returns nil, etc)

This shouldn't be the case with `@event['check']` and `@event['client']`. Those really should exist, and if they don't we have some other problem. I vote to avoid the indirection and just use the variables directly. They are just hashes and we can use them like normal. (cc @bobtfish in case he has an opinion, I think he originally wrote the method)